pkgs/servers/holochain-go/default.nix

```diff
@@ -0,0 +1,31 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "holochain-go${version}";
+  version = "v0.1.0-alpha";
```

Comment:
we usually do not include the `v` in the name, but rather do: `holochain-go-0.1.0-alpha`:

```nix
nix-repl> builtins.parseDrvName "holochain-gov0.1.0-alpha"
{ name = "holochain-gov0.1.0-alpha"; version = ""; }
nix-repl> builtins.parseDrvName "holochain-go-0.1.0-alpha"
{ name = "holochain-go"; version = "0.1.0-alpha"; }
```

Reply:
Thanks!